src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3465,6 +3531,34 @@ components:
             $ref: '#/components/schemas/SnapshotSummaryModel'
       description: >
         The total number of snapshots available that match the criteria and a page of summaries
+    EnumerateSnapshotSearchModel:
+      type: object
+      properties:
+        total:
+          type: integer
+          description: Total number of snapshots
+        filteredTotal:
+          type: integer
+          description: The number of snapshots included in this enumeration
+        items:
+          type: array
+          items:
+            $ref: '#/components/schemas/SnapshotSearchModel'
+      description: >
+        The total number of snapshots available that match the criteria and a page of summaries
+    SnapshotSearchModel:
+      type: object
+      properties:
+        snapshot:
+          $ref: '#/components/schemas/SnapshotModel'
```

Comment:
I don't think this will return the snapshot itself. I'm not sure it will be necessary to return the snapshot ID either, as the ID will be a field in the metadata in the result, so it can be derived from the metadata.

Reply:
what metadata model are you referring to?

Reply:
The one @fboulnois is working on. So in TIM this would be `dct:identifier` on the `TerraDCAT_ap:DataSnapshot` object. (It looks like TIM doesn't currently define `dct:identifier` for snapshots, but I think it wouldn't be controversial to add it.)